README.md

```diff
@@ -55,3 +55,70 @@ done in the template file, e.g.:
     </article>
 {% endfor %}
 ```
+
+You can add a side-bar listing documents with the same tags by including something like:
+
+```twig
+<aside> 
+{% for tag in meta.tags %}
+    <h3>Mere under: <a href="{{ base_url }}/tags/?tag={{ tag }}">{{ tag }}</a></h3>
```

Comment:
I would remove this line from the example to keep it minimal, even if it's a useful link.

Reply:
Nevermind, I realised it's working as a heading for each individual tag. Might be better to keep it 👍 
Maybe change the text to be in English instead though? "More pages tagged x" or something like that.

Reply:
I hadn't even noticed it :) I'll just make the changes online here, instead of pushing it from my clone, all right?

Reply:
That's perfectly fine by me!